README.md

```diff
@@ -380,6 +386,130 @@ jobs:
       working_directory: examples/my_project
 ```
 
+## Flutter Pub Publish Workflow
+
+### Steps
+
+The Flutter Pub Publish workflow consists of the following steps:
+
+1. Install Dependencies
+2. Setup pub credentials
+3. Dry run
+4. Publish
+
+### Inputs
+
+#### `flutter_channel`
+
+**Optional** The Flutter release channel to use (e.g. `stable`).
+
+**Default** `"stable"`
+
+#### `flutter_version`
+
+**Optional** The Flutter SDK version to use (e.g. `2.8.1`).
+
+**Default** `""`
+
+#### `working_directory`
+
+**Optional** The path to the root of the Flutter package.
+
+**Default** `"."`
+
+#### `runs_on`
+
+**Optional** An optional operating system on which to run the workflow.
+
+**Default** `"ubuntu-latest"`
+
+#### `pub_credentials`
+
+**Required** The pub credentials needed for publishing. This can be retrieved by reading out your `pub-credentials.json` on your system after you ran a `flutter pub login`, the location of the file is different per operating system:
```

Comment:
```suggestion
**Required** The pub credentials needed for publishing. This can be retrieved by reading out your `pub-credentials.json` on your system after you ran a `flutter pub login`. The location of the file is different per operating system:
```